superset-frontend/src/dashboard/components/menu/DownloadMenuItems/index.tsx

```diff
@@ -17,41 +17,53 @@
  * under the License.
  */
 import { Menu } from 'src/components/Menu';
-import DownloadAsImage from './DownloadAsImage';
-import DownloadAsPdf from './DownloadAsPdf';
+import { useToasts } from 'src/components/MessageToasts/withToasts';
+import DownloadScreenshot from './DownloadScreenshot';
+import { DownloadScreenshotFormat } from './types';
 
 export interface DownloadMenuItemProps {
   pdfMenuItemTitle: string;
   imageMenuItemTitle: string;
   addDangerToast: Function;
+  addSuccessToast: Function;
   dashboardTitle: string;
   logEvent?: Function;
+  dashboardId: string;
 }
 
 const DownloadMenuItems = (props: DownloadMenuItemProps) => {
   const {
     pdfMenuItemTitle,
     imageMenuItemTitle,
     addDangerToast,
-    dashboardTitle,
+    addSuccessToast,
     logEvent,
+    dashboardId,
     ...rest
   } = props;
 
+  const { addInfoToast } = useToasts();
+
   return (
     <Menu selectable={false}>
-      <DownloadAsPdf
+      <DownloadScreenshot
         text={pdfMenuItemTitle}
+        dashboardId={dashboardId}
         addDangerToast={addDangerToast}
-        dashboardTitle={dashboardTitle}
+        addSuccessToast={addSuccessToast}
```

Comment:
Can we just get addDangerToast, addSuccessToast and addInfoToast from `useToasts` called in `DownloadScreenshot` instead of passing them as props?